doc/api/util.md

```diff
@@ -371,6 +371,27 @@ added: v6.6.0
 A Symbol that can be used to declare custom inspect functions, see
 [Custom inspection functions on Objects][].
 
+## util.inspectTag
+<!-- YAML
+added: REPLACEME
+-->
+
+A template literal tag that ensures each replacement in a template string is
+passed through `util.inspect()`.
+
+```js
+const inspectTag = require('util').inspectTag;
+const obj = {a: 1};
+
+// Without the tag:
+console.log(`${obj}`);
+  // Prints: '[object Object]'
```

Comment:
Can we place these before the line of code it's referring to? To me that seems to be traditional place for comments. Also the indentation makes it look out of place.